### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by PavlovaOlena**
[DEMO LINK](https://PavlovaOlena.github.io/layout_stop-watch/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,20 @@
...
+    <title>Stop watch</title>
+    <link rel="stylesheet" href="./styles/main.scss">
+  </head>
+  <body>
+    <div class="clock">
+      <div class="clock__arrow-minute"></div>
+      <div class="clock__arrow-second"></div>
+      <div class="clock__center">
+
+      </div>
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/index.html:17]:
```suggestion
      <div class="clock__center"></div>
```

File: src/styles/blocks/clock.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
+.clock {
+  position: relative;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/clock.scss:2]: do not set position or margins to the block

Comment (Volodymyr-Melnyk-Fron10der) [reply src/styles/blocks/clock.scss:2]: use mixes

File: src/styles/blocks/clock.scss [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  &__arrow-second {
+    position: absolute;
+    border-radius: $radius-arrow;
+    width: $arrow-width / 2;
+    height: $arrow-height-second;
+    left: (($clock-size / 2) + $clock-space) - ($clock-center-size / 6);
+    top: $clock-space + ($arrow-height-minute - $arrow-height-second);
+    background-color: $arrow-second-bgc;
+    animation: clock 60s linear infinite;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/clock.scss:21]: move 60s into variable

File: src/styles/blocks/clock.scss [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+  }
+
+  &__arrow-minute {
+    position: absolute;
+    border-radius: $radius-arrow;
+    width: $arrow-width;
+    height: $arrow-height-minute;
+    left: (($clock-size / 2) + $clock-space) - ($clock-center-size / 3);
+    background-color: $arrow-minute-bgc;
+    animation: clock 3600s linear infinite;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/clock.scss:32]: into variable

File: src/styles/blocks/clock.scss [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  position: relative;
+  width: $clock-size;
+  height: $clock-size;
+  padding: $clock-space;
+  background-color: $clock-bgc;
+  margin: $clock-space auto;
+  border: 2px solid brown;
+  border-radius: $radius-circle;
+  transform-origin: bottom $center-dot;
+  transition: transform 3s;
```
Comment (Volodymyr-Melnyk-Fron10der) [inline src/styles/blocks/clock.scss:11]: set transition properties

**Review by Volodymyr-Melnyk-Fron10der — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (PavlovaOlena) [reply src/index.html]: Done

File: src/styles/blocks/clock.scss (hunk 2 above)
Comment (PavlovaOlena) [reply src/styles/blocks/clock.scss:2]: Done

File: src/styles/blocks/clock.scss (hunk 3 above)
Comment (PavlovaOlena) [reply src/styles/blocks/clock.scss]: Done

File: src/styles/blocks/clock.scss (hunk 4 above)
Comment (PavlovaOlena) [reply src/styles/blocks/clock.scss]: Done

File: src/styles/blocks/clock.scss (hunk 5 above)
Comment (PavlovaOlena) [reply src/styles/blocks/clock.scss]: Done

**Review by anstsot — APPROVED** [review]
Great job, approved!

**Pull request closed without merging**